using-timescaledb/data-retention.md

```diff
@@ -20,16 +20,38 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
-### Automatic Data Retention Policies
+### Automatic Data Retention Policies :enterprise_function:
 
-The `drop_chunks` command can be combined with an external tool for
-job scheduling, like `crontab` or `systemd`, to implement automatic
-data retention policies. Below we give some examples of how to
-implement such policies.
+TimescaleDB includes a background job scheduling framework for automating data
+management tasks, such as enabling easy data retention policies.
 
-Note that the path to `psql` and the parameters to connect to the
-database might differ depending on setup and need to be modified
-accordingly in the examples below.
+To add such data retention policies, a database administrator can create,
+remove, or alter policies that schedule `drop_chunks` to take place on a
+hypertable on a regularly scheduled interval.
+
+To add such a policy on a hypertable, simply execute the command:
+```sql
+SELECT add_drop_chunks_policy('conditions', interval '24 hours');
+```
+
+To subsequently remove the policy:
+```sql
+SELECT remove_drop_chunks_policy('conditions');
+```
+
+The scheduler framework also allows one to view scheduled jobs:
+```sql
+SELECT * FROM timescaledb_information.drop_chunks_policies;
+```
+
+For more information, please see the [API documentation](add_drop_chunks_policy).
```

Comment:
I think you need square brackets for the link name here.

Reply:
Updated.